src/components/folio-number-input/package.json

```diff
@@ -6,9 +6,9 @@
   },
   "dependencies": {
     "@bcrs-shared-components/interfaces": "^2.0.0",
-    "vue": "^2.7.14"
+    "vue": "^3.3.4"
```

Comment:
I think this component doesn't depend on (import) Vue any more, so this can be removed.

Reply:
Removed!